src/main/java/XmlParser.java

```diff
@@ -1,4 +1,41 @@
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+
 public class XmlParser {
+    // proszę o sprawdzenie
     public static void main(String[] args) {
+        String file = args[0];
+        parser(file);
+    }
+
+    public static String parser(String file) {
+        try {
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
```

Comment:
please use final var wherever it's possible 
https://developers.redhat.com/blog/2018/05/25/simplify-local-variable-type-definition-using-the-java-10-var-keyword